test/common-algorithms/dynamic-programming/sequential/fibonacci.chpl

```diff
@@ -0,0 +1,25 @@
+proc fib(n:int):int{
```

Comment:
There are actually several implementations of Fibonacci already in the test directory (grep for `fib`)